security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -0,0 +1,77 @@
++
+* In this release, the `nodeSelector` and `tolerations` in the `ScanSettings` object are now configurable  This enables deployers to configure where the Result Server will run, and thus what node will host the persistent volume that will contain the raw results. This allows us to schedule a pod that makes use of a persistent volume on the master nodes.
++
+* In this release, the Compliance Operator can now remediate `KubeletConfig` objects.
++
+* In this release, an enhancement had been added for content developers. An error is added to a comment for result objects that differentiates between objects that don't exist in the cluster versus objects that can't be fetched.
++
+* In this release, rule objects now contain two new attributes, `checkType` and `description`. These attributes allow the user to see if the rule pertains to a node check or to a platform check and also allows the user to audit what the rule check does.
++
+* In this release, the remediation template now allows multi-value variables. 
++
+* With this update, the Compliance Operator can change remediations based on variables that are set in the compliance profile. This is useful for remediations that include deployment-specific values such as time outs, NTP server host names, or similar. Additionally, the `ComplianceCheckResult` objects now use the label `compliance.openshift.io/check-has-value` that lists the variables a check can use.
++
+* This enhancement removes the requirement to extend an existing Profile in order to create a Tailored Profile. The `extends` field in the `TailoredProfile` CRD is no longer mandatory. You can now select a list of rule objects to create a tailor profile from scratch. Note: You will need to select whether your profile applies to nodes or the platform by setting the `compliance.openshift.io/product-type:` annotation or by setting the `-node` suffix for the `TailoredProfile` CR.
++
+* Previously, only master node taints were tolerated. Now, the Compliance Operator tolerates all taints, making it possible to schedule scans on all nodes.
+
+
+[id="openshift-compliance-operator-0-1-44-bug-fixes"]
+=== Bug fixes
+
+* Previously, while performing a scan, a crash was occurring in one of the scanner containers of the pods performing the scan. In this release, the Compliance Operator uses the latest OpenSCAP version 1.3.5 to avoid this issue.
++
+* Previously, only one global `KubeletConfig` object was created per pool. In this release, a `KubeletConfig` object is created for each pool. 
++
+* The rules for tailored profiles are now validated so that only rules of the same type (Platform or Node) are included in the validation.
```

Comment:
What issue was encountered?

Reply:
You know, on second though, this was a bug in a new functionality (creating the tailored profiles from scratch). So I don't think we need to remove it separately.
The issue was that while creating a new tailored profile from scratch, at one point CO wouldn't ensure that all the rules are of same type (all node rules or all platform rules). The issue was that the rules that were of different type than the profile itself would not be applicable and it was not apparent why. So to solve that, the operator now fails with a clear error message if the user tries to create a tailored profile from scratch and some of the rules don't match the rest.